ssl/internal.h

```diff
@@ -1118,12 +1118,27 @@ class SSLKeyShare {
 struct NamedGroup {
   int nid;
   uint16_t group_id;
-  const char name[8], alias[11];
+  const char name[16], alias[20];
 };
 
 // NamedGroups returns all supported groups.
 Span<const NamedGroup> NamedGroups();
 
+// PQGroup and PQGroups() are small helpers to encapsulate and keep track of
+// the component members of the hybrid named group.
+struct PQGroup {
```

Comment:
I guess I'm being lax with the naming, since in my mind it's sort of implied that (for the foreseeable future) any "PQ Group" will be inherently hybrid. I can be more explicit in the naming with something like `PQHybridGroups`.

Reply:
yeah, I think it'd be better to call them `PQHybridGroup`